src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SecureGroovyScriptTest.java

```diff
@@ -287,20 +284,16 @@
         for (File jarfile: getAllJarFiles()) {
             classpath.add(new ClasspathEntry(jarfile.getAbsolutePath()));
         }
-        
-        FreeStyleProject p = r.createFreeStyleProject();
-        p.getPublishersList().add(new TestGroovyRecorder(new SecureGroovyScript(
+        TestGroovyRecorder recorder = new TestGroovyRecorder(new SecureGroovyScript(
                 "whatever",
                 true,
                 classpath
-        )));
-        
-        JenkinsRule.WebClient wc = r.createWebClient();
-        r.submit(wc.getPage(p, "configure").getFormByName("config"));
-        
-        p = r.jenkins.getItemByFullName(p.getFullName(), FreeStyleProject.class);
-        TestGroovyRecorder recorder = (TestGroovyRecorder)p.getPublishersList().get(0);
-        assertEquals(classpath, recorder.getScript().getClasspath());
+        ));
+        TestGroovyRecorder recorder2 = r.configRoundtrip(recorder);
+        r.assertEqualDataBoundBeans(recorder, recorder2);
+        classpath.clear();
+        recorder2 = r.configRoundtrip(recorder);
+        r.assertEqualDataBoundBeans(recorder, recorder2);
```

Comment:
This just verifies that the normal case of empty classpaths works in the normal usage mode from, say, Groovy Postbuild. Does not attempt to reproduce bug in Promoted Builds, which would require (for example) a separate mock publisher that has an initially nonempty `List<SecureGroovyScript>`.